src/System.Runtime.WindowsRuntime.UI.Xaml/src/Resources/System.Runtime.WindowsRuntime.UI.Xaml.rd.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Directives xmlns="http://schemas.microsoft.com/netfx/2013/01/metadata">
+  <Library Name="System.Runtime.WindowsRuntime.UI.Xaml">
+    <!-- System.Private.Interop will create the following type through reflection, see GetMappingExceptionForHR for more info-->
+    <Type Name="Windows.UI.Xaml.Markup.XamlParseException" Dynamic="Required All" />
```

Comment:
Can you narrow these to Activate="Required Public"? We shouldn't need the rest of these members to be reflectable.